chore: Change GPIO library to libgpiod (#42)
canonical/matter-pi-gpio-commander: README.md

Usage instructions are available below and on the **[wiki](https://github.com/canonical/matter-pi-gpio-commander/wiki)**.

## Hardware Compatibility

> Until now, this snap will work on the following hardware:

Reviewer: ```suggestion
This snap is expected to work on the following hardware:
```
